ceremonies/src/lib.rs

```diff
@@ -76,18 +76,12 @@ pub mod pallet {
 		+ encointer_scheduler::Config
 	{
 		type Event: From<Event<Self>> + IsType<<Self as frame_system::Config>::Event>;
+
+		type CeremonyMaster: EnsureOrigin<Self::Origin>;
+
 		type Public: IdentifyAccount<AccountId = Self::AccountId>;
 		type Signature: Verify<Signer = Self::Public> + Member + Decode + Encode + TypeInfo;
 		type RandomnessSource: Randomness<Self::Hash, Self::BlockNumber>;
-		// The number of ceremony cycles that a participant's reputation valid for
-		#[pallet::constant]
-		type ReputationLifetime: Get<u32>;
-		// The number newbies a bootstrapper can endorse to accelerate community growth
-		#[pallet::constant]
-		type EndorsementTicketsPerBootstrapper: Get<u8>;
-		// The number of ceremony cycles a community can skip ceremonies before it gets purged
-		#[pallet::constant]
```

Comment:
I would like to keep this documentation for the storage values.

Also, please change them to `///` doc strings.